File: loginPages/home_login.py
import mysql.connector



# def main():

#     mydb = mysql.connector.connect(user='xliu95', password='50306778',host='oceanus.cse.buffalo.edu',database='cse442_2022_spring_team_n_db')

#     mycursor = mydb.cursor()
    
#     mycursor.execute("SELECT * FROM user;")
    
#     myresult = mycursor.fetchall()
    
#     for x in myresult:
#         print(x)
    
#     mydb.close()

def login(username,password):

    mydb = mysql.connector.connect(user='xliu95', password='50306778',host='oceanus.cse.buffalo.edu',database='cse442_2022_spring_team_n_db')

    mycursor = mydb.cursor()
    
    sql= "SELECT is_professor FROM user WHERE username =%s and password=%s"
    val = (username,password)
    mycursor.execute(sql,val)
    myresult = mycursor.fetchall()
    # return False if it is an inviald user
    if len(myresult) == 0:
        print("error")
        mydb.close()
        return (False,"Student")  
    else:
        # returns True and Student if the user is a student
        if myresult[0][0] == 0:
            print("student")
            mydb.close()
            return (True,"Student")
        # returns True and Professor if the user is a prof
        else:
            print("prof")
            mydb.close()
            return (False,"Professor")

    




# if __name__ == "__main__":
#     login("DemoStudent","DemoPass")
#     login("DemoProf", "DemoPass")
#     login("DemoStudent","WrongPass")

File: loginPages/home_login.php
<?php 

mysqli_report(MYSQLI_REPORT_ERROR | MYSQLI_REPORT_STRICT);

$mysqli = new mysqli("oceanus.cse.buffalo.edu", "johnbudn", "50382208", "cse442_2022_spring_team_n_db");

$username = "";
$password = "";
$login_err = "";
$result = "";
$col = "";

if($_SERVER["REQUEST_METHOD"] == "POST" && isset($_POST['submit'])){
    $mysqli = new mysqli("oceanus.cse.buffalo.edu", "johnbudn", "50382208", "cse442_2022_spring_team_n_db");  
    if (mysqli_connect_errno()) {
        echo "Failed to connect to MySQL: " . mysqli_connect_error();
        exit();
    }    
    // Check if username is empty   
    if (isset($_POST["username"]) && isset($_POST["password"])){
        $username = $_POST["username"]
        $password = $_POST["password"]
        print $username;
        print $password;

        $result = $mysqli->query("SELECT * FROM user where username = '$username' and password = '$password'");

        if($result->num_rows >0){
            echo "User validated";
            $col = $result->fetch_assoc();  

        if($col["is_professor"] == 0){
            echo " User is student";
        }
        else{
            echo " User is Professor";
        }
        else{
            echo "User not validated";
            $mysqli->close();
        }

    }
    else{
        $login_err = "invalid username or password"
    }
    // if(empty($_POST["username"])){
    //     $username_err = "Username cannot be empty.";
    // } else{
    //     $username = $_POST["username"];
    //     print $username;
    // }
    
    // // Check if password is empty
    // if(empty($_POST["password"])){
    //     $password_err = "Please cannot be empty.";
    // } else{
    //     $password = $_POST["password"];
    //     print $username;
    // }
    //   } 

}
